test/cmd/README.md

```diff
@@ -0,0 +1,196 @@
+# OpenShift Command-Line Integration Test Suite
+
+This document desceribes how OpenShift developers should interact with the OpenShift command-line integration test suite.
+
+## Test Strucure
+
+The script to run the entire suite lives in [`hack/test-cmd.sh`](./../../hack/test-cmd.sh). All of the test suites that make up
+the parent suite live in `test/cmd`, and are divided by functional area. 
+
+## Running Tests
+
+The full test suite can be invoked by running any of the following `make` targets, which run this suite among others,
+or the test runner script itself:
+```sh
+$ hack/test-cmd.sh
+$ make check-test
+$ make test-int-plus
+```
+
+Furthermore, any singular test suite can be run if an OpenShift instance is running and you are the cluster admin by invoking
+any of the scripts inside of `test/cmd`. The scripts will create a project for themselves to run inside of, and delete it after
```

Comment:
However, you can set that project up and they should all run.  Ie most
tests:

oc login ...
test/cmd/blah.sh

Is expected to work

On Dec 1, 2015, at 11:39 AM, Jordan Liggitt <notifications@github.com>
wrote:

In test/cmd/README.md
<https://github.com/openshift/origin/pull/6148#discussion_r46302296>:

> +
> +The script to run the entire suite lives in [`hack/test-cmd.sh`](./../../hack/test-cmd.sh). All of the test suites that make up
> +the parent suite live in `test/cmd`, and are divided by functional area.
> +
> +## Running Tests
> +
> +The full test suite can be invoked by running any of the following `make` targets, which run this suite among others,
> +or the test runner script itself:
> +```sh
> +$ hack/test-cmd.sh
> +$ make check-test
> +$ make test-int-plus
> +```
> +
> +Furthermore, any singular test suite can be run if an OpenShift instance is running and you are the cluster admin by invoking
> +any of the scripts inside of `test/cmd`. The scripts will create a project for themselves to run inside of, and delete it after

nope, they assume cluster-admin and a project they can use. caller sets
up/tears down. lets you create a project, run a script, and inspect results.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/6148/files#r46302296>.